syft/frameworks/torch/nn/conv.py

```diff
@@ -0,0 +1,111 @@
+import torch as th
+from torch.nn import Module
+
+
+class Conv2d(Module):
+    """
+    This class is the beginning of an exact python port of the torch.nn.Conv2d
+    module. Because PySyft cannot hook into layers which are implemented in C++,
+    our special functionalities (such as encrypted computation) do not work with
+    torch.nn.Conv2d and so we must have python ports available for all layer types
+    which we seek to use.
+
+    Note that this module has been tested to ensure that it outputs the exact output
+    values that the main module outputs in the same order that the main module does.
+
+    However, there is often some rounding error of unknown origin, usually less than
+    1e-6 in magnitude.
+
+    This module has not yet been tested with GPUs but should work out of the box.
```

Comment:
Might be worth capturing an issue for this.

Reply:
https://github.com/OpenMined/PySyft/issues/2927